openedx/core/djangoapps/api_admin/utils.py

```diff
@@ -11,5 +13,5 @@ def course_discovery_api_client(user):
     course_discovery_client = Client.objects.get(name=CLIENT_NAME)
     return EdxRestApiClient(
         course_discovery_client.url,
-        jwt=get_id_token(user, CLIENT_NAME)
+        jwt=get_id_token(user, CLIENT_NAME, secret_key=settings.JWT_AUTH['SECRET_KEY'])
```

Comment:
In general it would be good to plumb access to settings through the theming framework, because this will enable site-level overrides of that information -- it depends, though, on whether or not the setting is something that actually *should* be overridable per site.  We've been playing WAM with this, updating settings references as the need has come along for White Label work.